Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -150,10 +152,11 @@ WebIDL::ExceptionOr<void> HTMLElement::set_content_editable(StringView content_e
     return WebIDL::SyntaxError::create(realm(), "Invalid contentEditable value, must be 'true', 'false', or 'inherit'"_string);
 }
 
+// https://html.spec.whatwg.org/multipage/dom.html#set-the-inner-text-steps
```

Comment:
Can you add spec comments for the steps here?

Reply:
Ok, comments added.
The code is not exactly equal to the spec, but the code is equivalent, I put both steps above the code.